src/components/input-demo/index.tsx

```diff
@@ -8,17 +8,131 @@ export class SmoothlyInputDemo {
 		return [
 			<h2>Buttons</h2>,
 			<section>
+				<h4>Toggle button</h4>
+				<div style={{ display: "inline-block" }}>
+					<smoothly-toggle>
+						<smoothly-icon name="card" fill="solid" slot="icon-slot"></smoothly-icon>
+					</smoothly-toggle>
+					<smoothly-toggle>
+						<smoothly-icon name="briefcase" slot="icon-slot"></smoothly-icon>
+					</smoothly-toggle>
+					<smoothly-toggle>
+						<smoothly-icon name="airplane" slot="icon-slot"></smoothly-icon>
+					</smoothly-toggle>
+				</div>
+				<div style={{ display: "flex", marginTop: "1em" }}>
+					<smoothly-toggle disabled shape="rounded" name="This"></smoothly-toggle>
+					<smoothly-toggle shape="rounded" name="That"></smoothly-toggle>
+					<smoothly-toggle shape="rounded" name="Press"></smoothly-toggle>
+				</div>
+				<h4>Toggle switches</h4>
+				<div style={{ display: "inline-block", height: "3em", width: "6em", padding: "1em" }}>
+					<smoothly-toggle-switch disabled={true}></smoothly-toggle-switch>
+				</div>
+				<div style={{ display: "inline-block", height: "3em", width: "6em", padding: "1em" }}>
+					<smoothly-toggle-switch disabled={false}></smoothly-toggle-switch>
+				</div>
+				<div style={{ display: "inline-block", height: "2em", width: "4em", padding: "1em" }}>
+					<smoothly-toggle-switch disabled={false}></smoothly-toggle-switch>
+					<smoothly-toggle-switch disabled={false}></smoothly-toggle-switch>
+					<smoothly-toggle-switch disabled={false}></smoothly-toggle-switch>
+				</div>
+				<h4>Links with icons</h4>
+				<div style={{ display: "inline" }}>
```

Comment:
can maybe use span here which has default display inline.

Reply:
and in general. do not use inline styles. apply styles via CSS files.